### ISSUE TITLE
EM 3.0 testing with DAVID output

### ISSUE THREAD

**Issue by veroniquevoisin** [BaderLab/EnrichmentMapApp#265]
I have created a map from DAVID chart output. Once the map is generated the default labels (EM_GS_DESCR) are not useful as they only indicate the origin of the gene set (eg GOTERM_BP_ALL). We need to manually change the style to EM_Name which is better. However, EM_Name contains the gene set id followed by the description and separated by ~ GO:0006811~ION TRANSPORT. Would it be possible to automatically generate a new column from the DAVID output and set by default containing only the gene set description? (eg ION TRANSPORT)?
Thanks. Veronique

**Comment by veroniquevoisin** [comment]
![screen shot 2017-08-24 at 1 16 30 pm](https://user-images.githubusercontent.com/7570379/29679971-24de740e-88d1-11e7-8ff2-d8c4b1dc2fc1.png)

**Comment by veroniquevoisin** [comment]
![screen shot 2017-08-24 at 1 20 05 pm](https://user-images.githubusercontent.com/7570379/29679983-2e5b5ff6-88d1-11e7-8230-b6904ca9c616.png)

**Comment by veroniquevoisin** [comment]
I have tried on Em2.2.1 and it was worse as I could not even change the label.  I has to change the visual style to default and reconstruct the style. I had to sparse the DAVID data manually and import is as an attribute table for the label names . I had to uncheck case sensitive as it looks like EM transformed the lower cases into upper case. I set '%' from the DAVID output style as color. 
![screen shot 2017-08-24 at 3 39 34 pm](https://user-images.githubusercontent.com/7570379/29685130-81ccb3f4-88e2-11e7-91c4-37f70a69e8c1.png)
